common/persistence/cassandra/cassandraMetadataPersistenceProxy.go

```diff
@@ -112,23 +113,23 @@ func (m *metadataManagerProxy) UpdateDomain(request *p.UpdateDomainRequest) erro
 func (m *metadataManagerProxy) DeleteDomain(request *p.DeleteDomainRequest) error {
 	err := m.metadataMgr.DeleteDomain(request)
 	if err != nil {
-		m.logger.Warnf("Error deleting domain from V1 table: %v", err)
+		m.logger.Warn("Error deleting domain from V1 table", tag.Error(err))
```

Comment:
This is such a common occurrence. May be introduce something like WarnErr(string, error, fields...) to wrap this?

Reply:
Better not. Firstly I don't think this is very common. A lot of them also have other tags.
Secondly I will also need to fix the callat tag if introducing such helper. We already had similar discussion about more helpers, like LifeCycle helpers and the answer is no.